api/app/events/api.py

```diff
@@ -7,26 +7,33 @@
 from sqlalchemy.exc import SQLAlchemyError
 
 from app.events.models import Event, EventRole
-from app.events.mixins import EventsMixin
+from app.events.mixins import EventsMixin, EventsKeyMixin
 from app.users.models import AppUser
 from app.users.repository import UserRepository as user_repository
 from app.applicationModel.models import ApplicationForm
 from app.responses.models import Response
 
 from app import db, bcrypt, LOGGER
-from app.utils.errors import EVENT_NOT_FOUND, FORBIDDEN
+from app.utils.errors import EVENT_NOT_FOUND, FORBIDDEN, EVENT_WITH_KEY_NOT_FOUND
 
 from app.utils.auth import auth_optional, auth_required
 from app.utils.emailer import send_mail
+from app.events.repository import EventRepository as event_repository
+from app.organisation.models import Organisation
 
 
-def event_info(user_id, event):
+def event_info(user_id, response):
```

Comment:
Would change the name here to avoid confusion - took me a while to figure out it wasn't an HTTP response, but rather one of the named tuples that come back from the repository :D Maybe `event_org` ?

Reply:
Yes, that's better. I have changed the name